### PR TITLE
4 accommodations new acc page

### PR DESCRIPTION
**Pull request by OneInA1E6**
Added retrieving data from the database to display on accommodations page.

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+	<name>Capstone</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+		<nature>org.jkiss.dbeaver.DBeaverNature</nature>
+	</natures>
+</projectDescription>
```
Comment (mvallido) [inline .project:12]: We don't need the dbeaver file

File: .project [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>Capstone</name>
+	<comment></comment>
+	<projects>
```
Comment (KentonMewling) [inline .project:5]: Remove this

File: src/routes/web.php [hunk 3]
```diff
@@ -30,11 +30,13 @@
     return Inertia::render('Dashboard');
 })->middleware(['auth', 'verified'])->name('dashboard');
 
+
+
+Route::get('/accommodations', [Controllers\AccommodationController::class, 'show'])
```
Comment (KentonMewling) [inline src/routes/web.php:35]: looks good

File: src/resources/js/Pages/Accommodations/Index.vue [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+</template>
+
+<script setup>
+    import AuthenticatedLayout from '@/Layouts/AuthenticatedLayout.vue';
+    import SearchBar from '@/Components/SearchBar.vue'
+    import { Head } from '@inertiajs/inertia-vue3';
+
+    import { ref } from "vue";
+    let input = ref("");
+    const fruits = ["apple", "banana", "orange"];
```
Comment (KentonMewling) [inline src/resources/js/Pages/Accommodations/Index.vue:43]: weird but good job

File: src/database/seeders/UserSeeder.php [hunk 5]
```diff
@@ -16,10 +16,14 @@ class UserSeeder extends Seeder
     public function run()
     {
         for ($i = 1; $i <= 5; $i++) {
-        User::factory()->create([
+            User::factory()->create([
                     'email' => 'agent'.$i.'@gems.com'
             ]);
 
-           }
```
Comment (KentonMewling) [inline src/database/seeders/UserSeeder.php:23]: good work on the factory

**Review by KentonMewling — APPROVED** [review]

**Review by mvallido — CHANGES_REQUESTED** [review]
Just need to remove the dbeaver file

**Review by mvallido — APPROVED** [review]

**Pull request merged**